src/datanode/Cargo.toml

```diff
@@ -44,6 +45,7 @@ object-store = { path = "../object-store" }
 pin-project = "1.0"
 prost.workspace = true
 query = { path = "../query" }
+regex = "1.6"
```

Comment:
Do we still need this crate? I found that you remove the `SCHEMA_PATTERN` below.

Reply:
> Do we still need this crate? I found that you remove the `SCHEMA_PATTERN` below.

Yep, we still need the regex crate.

https://github.com/GreptimeTeam/greptimedb/blob/3a527c0fd59f5e98eda7abef5c54f03bfab0c22f/src/datanode/src/sql/copy_table_from.rs#L146-L151